README.md

```diff
@@ -8,12 +8,12 @@ GitHub Action to run [VHS][vhs].
 
 ## Inputs
 
-Name            | Description                | Default
---------------- | -------------------------- | ---------------------
-`path`          | Path of the VHS .tape file | ``"vhs.tape"``
-`version`       | Version of VHS to use      | ``"latest"``
-`token`         | GitHub token to use        | ``"${{ github.token }}"``
-`install-fonts` | Whether to install fonts   | ``"true"``
+Name                  | Description                      | Default
+---------------       | --------------------------       | ---------------------
+`path`                | Path of the VHS .tape file       | ``"vhs.tape"``
+`version`             | Version of VHS to use            | ``"latest"``
+`token`               | GitHub token to use              | ``"${{ github.token }}"``
+`install-fonts`       | Whether to install extra fonts   | ``"false"``
```

Comment:
```suggestion
`install-extra-fonts`       | Whether to install extra fonts   | ``"false"``
```
Did we change this?

Reply:
I think keeping `install-fonts` is the right approach to keep backward compatibility.